NOTICE

```diff
@@ -48,7 +48,7 @@ Matt Robenolt <mattr@disqus.com>
 Phillip Couto <phillip.couto@stemstudios.com> *
 Niklas Korz <korz.niklask@gmail.com>
 Nimi Wariboko Jr <nimi@channelmeter.com>
-Ghais Issa <ghais.issa@gmail.com>
+Ghais Issa <ghais.issa@gmail.com> *
```

Comment:
Technically, changes in this file are not related to `CONTRIBUTING.md`, I would prefer if it was a separate commit.

Reply:
done, in https://github.com/apache/cassandra-gocql-driver/pull/1792

Reply:
Thank you! There are still two other entries in NOTICE changed by this pull request.

Reply:
Good catch, thanks.  I got a bit confused bringing all the changes over to https://github.com/apache/cassandra-gocql-driver/pull/1792.  Fixed now.